src/runtime_src/core/tools/common/XBHelpMenusCore.cpp

```diff
@@ -557,3 +558,53 @@ XBUtilities::report_subcommand_help( const std::string &_executableName,
       std::cout << fmtExtHelp % formattedString;
   }
 }
+
+std::vector<std::string>
+XBUtilities::process_arguments( po::variables_map& vm,
+                                po::command_line_parser& parser,
+                                const po::options_description& options,
+                                const po::positional_options_description& positionals,
+                                bool validate_arguments
+                                )
+{
+  // Add unregistered "option"" that will catch all extra positional arguments
+  po::options_description _options(options);

```

Comment:
We are trying to get away from using underscore in front of variable names.  

Side Note
Currently, they are being used in parameter variable names, but we are now frowning upon that when when possible it is being removed.